### ISSUE TITLE
Supporting the new chat completion request? :)

### ISSUE THREAD

**Issue by bbence84** [Azzeccagarbugli/openai_client#9]
Hi!
Do you plan to support the just released ChatGPT chat completion request type?
https://platform.openai.com/docs/guides/chat

Thanks a lot! :)
Bence

**Comment by Azzeccagarbugli** [comment]
Hey @bbence84, my plan was to implement this brand-new update this weekend.

Hopefully, it should be straightforward 🤙

**Comment by bbence84** [comment]
Awesome, thanks! Actually I couldn't wait so I did a quick and dirty fork, which seems to work fine:
https://github.com/Azzeccagarbugli/openai_client/compare/main...bbence84:openai_client:bbence84-chatcompletion
I don't want to do a pull request since it might not be the nicest way to add it your library, and e.g. the new message structure of the new endpoint is only declared as a dynamic map, not an actual object... Also, I might have missed a few new paramters of the new endpoint...
But at least I confirmed that it works. :)
